plugins/base/src/main/kotlin/parsers/MarkdownParser.kt

```diff
@@ -159,17 +161,37 @@ class MarkdownParser(
 
         private fun linksHandler(linkText: ASTNode, link: String, linkTitle: ASTNode? = null): DocTag {
             val dri: DRI? = resolveDRI(link)
+            val linkTextString =
+                if (linkTitle == null) link else text.substring(linkTitle.startOffset + 1, linkTitle.endOffset - 1)
+
             val params = if (linkTitle == null)
                 mapOf("href" to link)
             else
-                mapOf("href" to link, "title" to text.substring(linkTitle.startOffset + 1, linkTitle.endOffset - 1))
+                mapOf("href" to link, "title" to linkTextString)
 
-            return DocTagsFromIElementFactory.getInstance(
-                MarkdownElementTypes.INLINE_LINK,
-                params = params,
-                children = linkText.children.drop(1).dropLast(1).evaluateChildren(),
-                dri = dri
-            )
+
+            return if (dri == null && !link.isRemoteLink()) {
+                DocTagsFromIElementFactory.getInstance(
+                    MarkdownTokenTypes.TEXT,
+                    params = params,
+                    children = linkText.children.drop(1).dropLast(1).evaluateChildren(),
+                    body = linkTextString.removeSurrounding("[", "]")
+                )
+            } else {
+                DocTagsFromIElementFactory.getInstance(
+                    MarkdownElementTypes.INLINE_LINK,
+                    params = params,
+                    children = linkText.children.drop(1).dropLast(1).evaluateChildren(),
+                    dri = dri
+                )
+            }
+        }
+
+        private fun String.isRemoteLink() = try {
+            URL(this)
```

Comment:
I was looking for some more straight-forward way of checking this. 
@Kordyjan Do you know a better method? 🙄

Reply:
I think this is the way many people do this. It's not the nicest API, but seems to work for others as well 🤷